docs/readthedocs/source/doc/Nano/Howto/accelerate_pytorch_inference_onnx.md

```diff
@@ -0,0 +1,55 @@
+# Accelerate PyTorch Inference using ONNXRuntime
+
+You can use ``Trainer.trace(accelerator='onnxruntime')`` API to enable the ONNXRuntime acceleration for PyTorch inference. It only takes a few lines.
+
+Let's take an [ResNet-18 model](https://pytorch.org/vision/main/models/generated/torchvision.models.resnet18.html) pretrained on ImageNet dataset as an example. First, we load the model:
+
+```python
+from torchvision.models import resnet18
+
+model_ft = resnet18(pretrained=True)
+```
+
+Then we set it in evaluation mode:
+
+```python
+model_ft.eval()
```

Comment:
shall we move this step to `Trainer.trace`? @TheaperDeng

Reply:
What if the user does not set `eval`? I think we should automatically set it to `eval` in `trace` and then restore the status.